docs/index.rst

```diff
@@ -29,6 +29,8 @@ Pymagicc
     pymagicc.api
     pymagicc.io
     pymagicc.definitions
+    pymagicc.config
+    pymagicc.utils
```

Comment:
```suggestion
```
Remove module with only private functions

Reply:
It claims that you could add a commit with this directly ... can see a button to do this?

Reply:
Or alternatively one could add something general to `pymagicc.utils.rst`